### ISSUE TITLE
Why 2 TXT records?

### ISSUE THREAD

**Issue by rb07** [jeansergegagnon/zoneedit_letsencrypt#14]
Hi & thanks for your scripts.

I was trying to renew my certificate and it failed.  I went to check the SPF TXT record which seemed to be causing the error you mention but I saw 2 TXT _acme-challenge records, which seems odd but I thought perhaps I left an old one.

I deleted both of those, kept my SPF and retried.  This time the script worked fine.

The question is a the subject above, why create 2 different records?  Or more to the point, why is one of those making the script fail.

Actually the log did show exactly that:
```
   Domain: sample.org
   Type:   unauthorized
   Detail: Incorrect TXT record
   "37dOpkVV_Sz719tlbsRbd6NrVyS_UwlwETOxWFhDm88" (and 1 more) found at
   _acme-challenge.sample.org
```
That was the first record, the second had exactly what the actual acme-challenge was.

**Comment by jeansergegagnon** [comment]
Thanks for question.

I'm not really sure why there would be 3 TXT records, I'll have to check.

Do you think it's my script that created the 2nd record?

**Comment by rb07** [comment]
Yes.  There are 2 new records after successfully running the script.

**Comment by jeansergegagnon** [comment]
Oh, the 2 TXT acme are because there's two verifications done by
letsencrypt - I'm not sure if they both need to stay there, but I've always
kept the 2 there myself.

As for the SPF issue I mentioned, I thought it was fixed although you say
you had an issue, so maybe not.

Let me know if you still have issue next time to renew

**Comment by rb07** [comment]
OK, thanks.

If I remember correctly, only the first time there is a need for 2 verifications, but when I've done it manually I just changed the single record.  Also there should be no problem with several TXT records, but I don't remember seeing it in use before.  What seems important is the log message.

There were no SPF issues as far as I can see.  The html output shows that message but I think its part of the page, not the actual error message.
